zooniverse/front-end-monorepo
# Create `lib-subject-viewers` & make importable for `lib-classifier`

## Package

`lib-subject-viewers`

## Linked Issue and/or Talk Post
n/a

## Describe your changes
- Create the base structure for the `lib-subject-viewers` package in FEM. 
- Make sure `lib-classifier` is able to import the package

## How to Review
1) Run `yarn panic && yarn bootstrap` to create a fresh codebase
1) Run all commands in the `package.json` to ensure they all succeed and none error out
1) Check that the ProtoViewer component renders properly in Storybook
1) Test adding the component to the `lib-classifier` package. Go into the `lib-classifier/dev/components/App.js` file and import the `ProtoViewer` at the top and then test that it renders properly in the component.

```js
import { ProtoViewer } from '@zooniverse/subject-viewers'
```

## General
